src/DenielWorld/VanillaCommands/command/Connect.php

```diff
@@ -6,28 +6,31 @@
 use pocketmine\command\CommandSender;
 use pocketmine\command\PluginCommand;
 use pocketmine\command\PluginIdentifiableCommand;
-use pocketmine\command\utils\InvalidCommandSyntaxException;
 use pocketmine\Player;
+use pocketmine\utils\TextFormat as TF;
 
 class Connect extends PluginCommand implements PluginIdentifiableCommand{
 
     public function __construct(string $name, Loader $owner)
     {
         parent::__construct($name, $owner);
-        $this->setUsage("/connect <serverUri: text>");
-        $this->setDescription("Attempts to connect to the websocket server on the provided URL");
+        $this->setUsage("/connect <Ip> <Port></Port>");
+        $this->setDescription("Connect to other servers");
```

Comment:
uhm.. wasn't that explicitly changed because it was incorrect? You must decide if you want vanilla commands, or custom shit.

Reply:
As I stated in my comment, I'm not touching connect in this PR.

Reply:
Couldn't see any comment, but a question if it is a packet or not

Reply:
Didn't see one